docs/components/operate/deployment/configuration.md

```diff
@@ -109,7 +130,7 @@ camunda.operate:
     # Cluster name
     clusterName: elasticsearch
     # Url
-    url: http://localhost:9200
+    url: https://localhost:9200
```

Comment:
don't we need `selfSigned` here too?

Reply:
As I tested it on my box, I didn't need it.I imported the certificate into JVM runtime with `keytool` and then it works just by specifying the `https` as protocol.